recipes-core/images/core-image-minimal.bbappend

```diff
@@ -7,6 +7,8 @@ IMAGE_INSTALL_append = " \
    packagegroup-rte-stm \
    packagegroup-rte-coreboot \
    locale-base-en-us \
+   python3-setuptools \
```

Comment:
@TomaszAIR setuptools nie powinno być potrzebne na obrazie docelowym.

Reply:
bez setuptools otrzymywałem output
```
    from pkg_resources import load_entry_point
ImportError: No module named pkg_resources
```
Sprawdzę to jeszcze jak skończę obecne zadanie.

Reply:
@TomaszAIR Przy uruchamianiu `wakeonlan`? Jeśli jakiś pakiet jest potrzebny do działania innego, to dodajemy to w RDEPENDS w recipe tego pakietu, a anie obrazu.https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html